src/components/StructuredData.astro

```diff
@@ -0,0 +1,71 @@
+---
+import * as prismicH from "@prismicio/helpers";
```

Comment:
Shouldn't these be single quotes (just from looking at other files)? 

I also noticed a tabs vs spaces inconsistency in a different file.
![](https://user-images.githubusercontent.com/9084735/205178661-d8e4d3cb-2657-4e6b-bf2d-bc6a1df0f836.png)

And I had similar formatting issues in #26 too. This repo could probably benefit from an explicit prettier config.

Reply:
Some of the other files have double-quotes for imports 😅 - I'm happy to go with either way

Reply:
Corrected the indentation